src/App.tsx

```diff
@@ -152,8 +178,27 @@ export default function App() {
             Calculate
           </Button>
         </Grid>
-        <h2>Results:</h2>
-        {results && <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />}
+        <Grid container>
+          <Grid container item xs={6} direction="row">
+            <div>
+              <h2>Results:</h2>
+              {results && (
+                <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
+              )}
+            </div>
+          </Grid>
+          <Grid container item xs={6} direction="row" wrap="nowrap">
+            {htmls &&
+              htmls.map(html => {
+                return (
+                  <div key={html.groupId}>
+                    <h2>HTML:</h2>
+                    <h6>{parse(html.html)}</h6>
```

Comment:
```suggestion
                    {parse(html.html)}
```

We shouldn't wrap this in an h6, we can just inject the raw html from the parse into the DOM.